RELEASE_NOTES.md

```diff
@@ -1,5 +1,98 @@
 # Release Notes
 
+## 5.0.0
+### Features
+- VMs created by `podman machine` can now use the native Apple hypervisor (`applehv`) when run on MacOS.
+- A new command has been added, `podman machine reset`, which will remove all existing `podman machine` VMs and relevant configurations.
+- The default image to use with `podman machine` VMs can now be set from `containers.conf`.
+- The `podman manifest add` command now supports a new `--artifact` option which will create an artifact manifest including one or more files as OCI artifacts and add them to the specified manifest list.
```

Comment:
if this is accurate wording, then maybe we could rework this sentence to simplify it